seurat-find-transfer-anchor.R

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env Rscript
+
+suppressPackageStartupMessages(require(optparse))
+suppressPackageStartupMessages(require(workflowscriptscommon))
+
+#set parsing options
+
+option_list = list(
+ 
+  
+   make_option(
+    c("-i", "--query-file"),
+    action = "store",
+    default = NA,
+    type = 'character',
+    help = "seurat object to use as the reference",
```

Comment:
You need to tweak this to reflect our command-line usage. So this would be 'File containing seurat object to use as the reference'. Also, please capitalise first character of all help strings.